app/Models/Batch/Batches.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Models\Batch;
+
+use App\Models\Site\Municity;
+use App\Models\Site\Province;
+use App\Models\Site\Region;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Orchid\Filters\Filterable;
+
+class Batches extends Model
```

Comment:
Model class name should be in singular form: `Batch`.

Reply:
Changed Batches model to 'Batch'. Also changed it to screen referencing to the model.